com.unity.render-pipelines.universal/Runtime/Debug/DebugDisplaySettingsMaterial.cs

```diff
@@ -251,7 +222,7 @@ internal static class WidgetFactory
                 setter = (value) => {},
                 getIndex = () => (int)data.albedoDebugValidationPreset,
                 setIndex = (value) => data.albedoDebugValidationPreset = (AlbedoDebugValidationPreset)value,
-                onValueChanged = (field, value) => DebugManager.instance.ReDrawOnScreenDebug()
+                onValueChanged = (_, _) => DebugManager.instance.ReDrawOnScreenDebug()
```

Comment:
normally for unused parameters I think is better to put `(_, __)`

Reply:
I think in order to make it a [C#7 discard](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards), the single underscore is what I need to use. My guess is that before C#7, this was more of a common naming convention and hence you couldn't use duplicate variable names.

Reply:
maybe I am becoming and old guy :)